tools/klockstat.py

```diff
@@ -363,8 +363,8 @@ def display(sort, maxs, totals, counts):
             stack  = list(stack_traces.walk(k.value))
             caller = b.ksym(stack[1], show_offset=True)
 
-            if (args.caller and caller.find(args.caller)):
-                continue
+        if (args.caller and caller.find(args.caller.encode())):
```

Comment:
I think we should keep the check under the current condition,
because we don't know what data we missed and omitting
we could mislead the user